packages/shared/src/graphql/prompt.ts

```diff
@@ -14,6 +15,20 @@ export type Prompt = {
   flags?: PromptFlags;
 };
 
+export enum PromptDisplay {
```

Comment:
This can't really be a enum right considering they are dynamic and will change over time.